frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -222,7 +222,7 @@ action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message.HTML=
 
 
 action.GitMachete.TraverseFromSelectedAction.text=Git Machete: Traverse from Selected Branch\u2026
-action.GitMachete.TraverseFromSelectedAction.GitMacheteContextMenu.text=T_raverse from Selected Branch\u2026
+action.GitMachete.TraverseFromSelectedAction.GitMacheteContextMenu.text=_Traverse from here\u2026
```

Comment:
Uppercase `Here`? dunno

Reply:
no!

Reply:
oh, ok! that's how it looks like in git4idea

Reply:
capitalised `Here` 👍 but regarding to Slide In / Slide Out I would leave it as it is b/c it's like the preposition is a part of the action name in that case